components/gestion-admin/administrateurs.js

```diff
@@ -0,0 +1,89 @@
+import {useState, useEffect, useCallback, useContext} from 'react'
+import {orderBy} from 'lodash'
+
+import {getAdministrators, addAdministator} from '@/lib/suivi-pcrs.js'
+
+import AuthentificationContext from '@/contexts/authentification-token.js'
+
+import AdministrateurList from '@/components/gestion-admin/administrateur-list.js'
+import Header from '@/components/gestion-admin/header.js'
+import Loader from '@/components/loader.js'
+
+const Administrateurs = () => {
+  const {token, isTokenRecovering} = useContext(AuthentificationContext)
+
+  const [admins, setAdmins] = useState([])
+  const [filteredAdmins, setFilteredAdmins] = useState([])
+  const [isLoading, setIsLoading] = useState(false)
+  const [validationMessage, setValidationMessage] = useState(null)
+  const [errorMessages, setErrorMessages] = useState({headerError: null, fetchError: null})
+
+  const getAdmins = useCallback(async () => {
+    setIsLoading(true)
+    setErrorMessages(errorMessages => ({...errorMessages, fetchError: null}))
+
+    try {
+      const administrators = await getAdministrators(token)
+
+      setAdmins(orderBy(administrators, [item => item.nom ? item.nom.toLowerCase() : 'N/A'.toLowerCase()], ['asc']))
```

Comment:
On perd en lisibilité, je te propose de modifier un peu.

```suggestion
      const orderedAdministrators = orderBy(administrators, [item => item.nom ? item.nom.toLowerCase() : 'N/A'.toLowerCase()], ['asc']) // Order by name
      setAdmins(orderedAdministrators)
```

Reply:
J'ai une fonction permettant de gérer ce cas que je n'avais pas intégré à cet endroit. L'outil `normalizeSort()` permettant de traiter les chaînes de caractères dans une fonction `orderBy` sera utilisée ici